### ISSUE TITLE
[UX] Pull-to-refresh gesture to refresh cache details

### ISSUE THREAD

**Issue by Lomanic** [cgeo/cgeo#8888]
**Is your feature request related to a problem?**
Drop-down menu in cache page is hard to use as it has many entries with only text labels and the action to refresh a cache is really common on a cache details page. 

**Describe the solution you'd like**
Implement a "pull-to-refresh" gesture to refresh cache details. [It can be implemented with AndroidX](https://stackoverflow.com/a/22786827) that cgeo already uses. It would greatly improve UX, as this is one of the most (if not the most) used entries in the cache details menu, and this gesture is now common and expected (the official GC app implements it on the cache details view for example). Also it doesn't conflict with any existing gesture in the app. 

**Describe alternatives you've considered**
This should probably be made a separate dedicated feature request, but prepending icons for each actions of the cache details drop-down menu would greatly improve UX, this is something I think about for years now. 

**Additional context**
 If this is to be implemented, maybe we could add this pull-to-refresh gesture to search results lists as when you go back and forth between cache details and search results (or you go back to cgeo after a while), you can lose all the results not in local cache. There are maybe other views where this pattern is applicable.

**Comment by rsudev** [comment]
Did you know that there is a dedicated refresh button on the cache details page?

**Comment by Lomanic** [comment]
Of course. Let me more explicit of my usual use case(s)

I'm on the logs panel, looking at some logs. I want to update them for any of the following reasons:

* local data is suspected to be old, maybe there are new logs and I'm looking for hints on the field
* I sent a new log and want to see this log as rendered on the website (smilies and formatting, as this is not done by cgeo locally) and I want to be sure `[NUMBER]` was expanded to the right value
* I edited the log on the website (as unfortunately cgeo doesn't permit this, yet?) and want to have it updated locally also
* I've sent a log for this cache through GC website on desktop and want to have it also in cgeo

Currently to achieve this I have to follow one of these two paths:

* go back to the initial details panel where the update button is (two swipes), clicking this button, go back to the logs panel (two swipes)
* stay on the logs panel, click to show the menu, then find the Refresh entry (not straightforward as written before) and click on it

Another use case (but rare) is with the Description panel: if there is a dynamically generated image embedded (in Mysteries) that changes on each request, we may want to refresh this panel repeatedly without hassle. 

If this pull-to-refresh gesture was implemented, updating cache details one would just swipe from top to bottom.
